samples/dynamodb/lambda/sample.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Using NServiceBus Sagas with AWS Lambda, SQS, and DynamoDB
+reviewed: 2023-05-09
+component: DynamoDB
+related:
+ - nservicebus/sagas
+ - samples/dynamodb/simple
+ - samples/hosting/aws-lambda-sqs
+---
+
+This sample shows a basic Saga using AWS Lambda with SQS and DynamoDB.
+
+downloadbutton
+
+## Prerequisites
+
+The sample includes a [`CloudFormation`](https://aws.amazon.com/cloudformation/aws-cloudformation-templates/) template, which will deploy the Lambda and create the necessary queues to run the sample.
+
+The [`Amazon.Lambda.Tools` CLI](https://github.com/aws/aws-lambda-dotnet) can be used to deploy the template to an AWS account.
+
+1. Install the [`Amazon.Lambda.Tools CLI`](https://github.com/aws/aws-lambda-dotnet#amazonlambdatools)
+1. Make sure an S3 bucket is available in the AWS region of choice
+
+## Running the sample
+
+INFO: It is not possible at this stage to use the AWS .NET Mock Lambda Test Tool to run the sample locally.
```

Comment:
I'd say we should either explain why or maybe just remove this note?

Reply:
I would guess it is a note we have elsewhere too don't we?

Reply:
Yes in the simple AWS Lambda sample.

Reply:
then we should explain why?